src/components/ResultList/Item/Item.tsx

```diff
@@ -126,3 +247,27 @@ export const Item = (props: IItemProps): ReactElement => {
     </article>
   );
 };
+
+interface IItemDropdownProps {
+  label: ReactElement | string;
+  items: ItemType[];
+}
+
+export const ItemDropdown = ({ label, items }: IItemDropdownProps): ReactElement => {
+  const viewport = useViewport();
+  const listClasses = viewport > Viewport.MD ? 'h-auto w-auto' : 'h-auto w-auto absolute top-full -right-0';
```

Comment:
If we're using this viewport calculation for just CSS, you should use tailwinds built-in utilities for that

https://tailwindcss.com/docs/responsive-design

Reply:
Fixed.